service-api/module/Application/src/DWP/AuthApi/DTO/ResponseDTO.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\DWP\AuthApi\DTO;
+
+class ResponseDTO
+{
+    public function __construct(
+        private readonly string $accessToken,
+        private readonly string $expiresIn,
+        private readonly string $tokenType,
+    ) {
+    }
+
+    public function accessToken(): string
+    {
+        return $this->accessToken;
+    }
+
+    public function expiresIn(): string
+    {
+        return $this->expiresIn;
+    }
```

Comment:
Similarly, you could make the properties `public readonly` rather than having a passthrough getter